grow/common/utils.py

```diff
@@ -279,6 +279,7 @@ def slugify(text, delim=u'-'):
     for word in SLUG_REGEX.split(text.lower()):
         word = word.encode('translit/long')
         if word:
+            word = "".join(SLUG_REGEX.split(word)) # deal with smartquotes
```

Comment:
Can you remove this from this PR? I'm not sure if that is how we want to solve that issue, and it helps to keep the PR cleaner to not have random other fixes in it.